docs/guide/contributing.rst

```diff
@@ -2,27 +2,27 @@
 Contributing to the Write the Docs guide
 =========================================
 
-If you want to contribute to an awesome documentation guide, you've come to the right place. 
-Welcome! Read on to understand what our guides are and how you can help.
+Welcome! The guides at Write the Docs are meant to address all aspects of software documentation. 
+To contribute to a guide, read on to understand our guides and how you can help.
 
 Ways to contribute
 -------------------
 
-We encourage contributions from anyone regardless of professional or tool experience. 
+Anyone can contribute regardless of professional or tool experience. 
 Contributions are treated with respect if they respect the current effort and state of 
 our guide.
 
-The guides at Write the Docs are meant to address all aspects of software 
-documentation. There are several ways to contribute to the content:
+There are several ways to contribute to the content:
 
+* :ref:`Edit a single guide directly in GitHub <update-guide-via-github>`.
 * For changes that span multiple files :ref:`create a fork, a branch, and a pull request <update-guide-in-git>`.
 * `Submit an issue`_ to suggest small changes, new content, or to file bugs against existing content.
 
-We encourage you to learn some of the tools we use, but if you don't want to deal with 
+We encourage you to learn some of the tools we use, but if you don't want to use 
 GitHub, email attachments or inline text to guide@writethedocs.org.
 
-Where to start
---------------
+Where to contribute
+--------------------
```

Comment:
```suggestion
What to contribute
-------------------
```

Reply:
Thanks for the recommendation. I was definitely considering to update and will do so since you are obviously in agreement.